--- /home/laura/Documents/AI/NLP/NLP_LM/NLP_LM/models/RAN/RAN.py
+++ /home/laura/Documents/AI/NLP/NLP_LM/NLP_LM/models/RAN/RAN.py
@@ -39,7 +39,7 @@
             hidden = F.tanh(latent)
             dropped = self.drop(hidden)
             output = self.h2o(dropped)
-            # output = F.log_softmax(output)
+            output = F.log_softmax(output)
             return content, latent, hidden, output
         else:
             temp_i = self.h2i(latent) + self.x2i(embeds)
@@ -49,7 +49,7 @@
             latent = torch.mul(input_gate, embeds) + torch.mul(forget_gate, latent)
             dropped = self.drop(latent)
             output = self.h2o(dropped)
-            # output = F.log_softmax(output)
+            output = F.log_softmax(output)
             return latent, hidden, output
 
     def initVars(self, cud, bsz):